src/AzSK.ADO/Framework/BugLog/BugLogCheckerHelper.ps1

```diff
@@ -0,0 +1,179 @@
+Set-StrictMode -Version Latest
+class BugLogCheckerHelper {
+
+    hidden static [BugLogCheckerHelper] $BugLogCheckerInstance;
+
+    hidden [bool] $UseAzureStorageAccount;
+    hidden [string] $OrganizationName;
+    hidden [string] $StorageAccount;
+
+    hidden [object] $StorageAccountCtx;
+    hidden [string] $StorageRG;
+    hidden [bool] $hasAccessOnStorage = $true;
```

Comment:
make it false and enable it only if access is found

Reply:
changed